packages/@yoda/util/fs.js

```diff
@@ -34,3 +35,23 @@ function mkdirp (dir, callback) {
     }) /** fs.stat */
   }) /** fs.mkdir */
 }
+
+/**
+ * @param {string} dir
+ */
+function mkdirpSync (dir) {
+  try {
+    fs.mkdirSync(dir)
+  } catch (err) {
+    if (err.code === 'ENOENT') {
+      return mkdirpSync(path.dirname(dir))
+    }
+    var stat = fs.statSync(dir)
+    if (!stat.isDirectory()) {
+      var eexist = new Error('Path exists')
+      eexist.path = dir
+      eexist.code = 'EEXIST'
+      throw eexist
+    }
+  }
```

Comment:
If this is a sync method, I'd tend to use `child process` instead, that means more performant.